### PR TITLE
Rename ConvertFToTF32INTEL to RoundFToTF32INTEL

### PR DESCRIPTION
**Pull request by MrSidims**
Extension name will be preserved for a while for binary compatibility.

There is not public spec change yet, but it has to be done ASAP

It's a cherry-pick of https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1913

### CODE REVIEW

File: llvm-spirv/lib/SPIRV/libSPIRV/spirv_internal.hpp [hunk 1]
```diff
@@ -101,6 +101,7 @@ enum InternalCapability {
   ICapFPArithmeticFenceINTEL = 6144,
   ICapGlobalVariableDecorationsINTEL = 6146,
   ICapabilityComplexFloatMulDivINTEL = 6414,
+  ICapabilityTensorFloat32RoundingINTEL = 6425,
   ICapabilityTensorFloat32ConversionINTEL = 6425,
```
Comment (asudarsa) [inline llvm-spirv/lib/SPIRV/libSPIRV/spirv_internal.hpp:105]: will the two enums having same value cause issues here? Also, may be a TODO here? Thanks

**Review by asudarsa — APPROVED** [review]
Looks good to me. Only one correction needed. Please address that before merging. Thanks

File: llvm-spirv/lib/SPIRV/libSPIRV/spirv_internal.hpp (hunk 1 above)
Comment (MrSidims) [reply llvm-spirv/lib/SPIRV/libSPIRV/spirv_internal.hpp:105]: oh, it's a cherry-pick mistake

**Comment by MrSidims** [comment]
Probably we don't need this patch ASAP here, @dkhaldi please confirm

**Comment by MrSidims** [comment]
Closing as we don't need it here ASAP and can wait for the pulldown

**Pull request closed without merging**